### PR TITLE
Uplift Kubernetes version to v1.23.5 and Kubenetes binaries config version to v0.13.0

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by Rozzii — DISMISSED** [review]

**Comment by Sunnatillo** [comment]
Kashif, can you please update k8s version here as well: 
https://github.com/Nordix/metal3-dev-tools/blob/7f1d68fa49df25b29922349b28ca39ffd355344f/ci/jobs/metal3_dev_tools_integration_tests.pipeline#L54

**Comment by kashifest** [comment]
> Kashif, can you please update k8s version here as well:
> 
> https://github.com/Nordix/metal3-dev-tools/blob/7f1d68fa49df25b29922349b28ca39ffd355344f/ci/jobs/metal3_dev_tools_integration_tests.pipeline#L54

Done

**Review by Sunnatillo — APPROVED** [review]
LGTM

File: ci/scripts/image_scripts/provision_node_image.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@ set -uex
 
 SCRIPTS_DIR="$(dirname "$(readlink -f "${0}")")"
 
-export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.23.3"}
+export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.23.5"}
 export KUBERNETES_BINARIES_VERSION="${KUBERNETES_BINARIES_VERSION:-${KUBERNETES_VERSION}}"
-export KUBERNETES_BINARIES_CONFIG_VERSION=${KUBERNETES_BINARIES_CONFIG_VERSION:-"v0.2.7"}
+export KUBERNETES_BINARIES_CONFIG_VERSION=${KUBERNETES_BINARIES_CONFIG_VERSION:-"v0.13.0"}
```
Comment (furkatgofurov7) [inline ci/scripts/image_scripts/provision_node_image.sh:9]: This looks like a big jump, so we were using the old binaries up until now?

**Review by furkatgofurov7 — DISMISSED** [review]
Looks good, can you please fix a typo (v1.25.5 to v1.23.5) in the PR title? Thanks

File: ci/scripts/image_scripts/provision_node_image.sh (hunk 1 above)
Comment (kashifest) [reply ci/scripts/image_scripts/provision_node_image.sh:9]: Yes

**Review by Sunnatillo — APPROVED** [review]
LGTM

**Pull request merged**